File: tests/XMP/LARGE-SIZE-TESTS/global-view/gmove/C/chk/chk.c
#include <stdio.h>
#include <stdlib.h>

int chk_int(int ierr){

#pragma xmp nodes p(*)

#pragma xmp task on p(1)
{
  if ( ierr == 0 ) {
     printf("PASS\n");
  }else{
     printf("ERROR\n");
     exit(1);
  }
}
  return 0;

}

File: tests/XMP/LARGE-SIZE-TESTS/global-view/gmove/F/chk/chk.f90
subroutine chk_int(ierr)
integer ierr
!$xmp nodes p(*)

!$xmp task on p(1)
  if ( ierr .eq. 0 ) then
     write(*,*) "PASS"
  else
     write(*,*) "ERROR"
     call exit(1)
  endif
!$xmp end task

return
end subroutine
